torch/csrc/autograd/python_variable_indexing.cpp

```diff
@@ -75,6 +75,10 @@ static Variable applySlice(const Variable& self, int64_t dim, PyObject* slice, b
     // TODO: implement negative step
     throw ValueError("negative step not yet supported");
   }
+  if (start >= stop) {
```

Comment:
Should this be `start == stop` instead of `start >= stop`?

Reply:
It should be effectively the same (since we don't support negative step)